src/response.ts

```diff
@@ -575,29 +575,54 @@ export class Response implements DenoResponse {
     return this.send(body);
   }
 
-  // TODO: back-compat support for Express signature.
-  // Namely objects and arrays.
   /**
    * Set header `field` to `value`, or pass
    * an object of header fields.
    *
    * Examples:
    *
-   *    res.set('Accept', 'application/json');
-   *
+   *     res.set('Accept', 'application/json');
```

Comment:
We should add an example of the array implementation as well!

```suggestion
   *     res.set('Accept', 'application/json');
   *     res.set('Foo', ['bar', 'baz']);
```

Reply:
Or perhaps a cookie example, a little more "real world"